server/jersey-server/src/main/java/com/quorum/tessera/server/JerseyServer.java

```diff
@@ -72,6 +72,8 @@ public void start() throws Exception {
         initParams.put("jersey.config.server.monitoring.statistics.enabled", "true");
         initParams.put("jersey.config.server.monitoring.enabled", "true");
         initParams.put("jersey.config.server.monitoring.statistics.mbeans.enabled", "true");
+        //custom properties
+        initParams.put("tessera.serverConfig", serverConfig);
```

Comment:
No.. We either access config in usual way . This second route to config and high jacking the initParams should be avoided